src/nvim/os/pty_process_unix.c

```diff
@@ -160,7 +160,6 @@ static void init_child(PtyProcess *ptyproc)
   os_unsetenv("COLUMNS");
   os_unsetenv("LINES");
   os_unsetenv("TERMCAP");
-  os_unsetenv("COLORTERM");
```

Comment:
hmm we should not just passively pass trough COLORTERM, but maybe it should be set to `truecolor` if 'termguicolors' is true and '256' otherwise.

Reply:
Done.